lib/menu.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'rice'
+require_relative 'chicken'
+require_relative 'lamb'
+require_relative 'mutton'
+require_relative 'naan'
+require_relative 'poppadom'
+# require_relative 'message'
+
+class Menu
+
+	attr_reader :items, :order, :total
+
+	def initialize
+		@items = []
+		@order = []
+		@total = []
+		items << Rice.new
+		items << Chicken.new
+		items << Lamb.new
+		items << Mutton.new
+		items << Naan.new
+		items << Poppadom.new
+	end
+
+	def display_menu
+		puts 'Rice           £2.35'
+		puts 'Chicken curry  £6.25'
+		puts 'Lamb curry     £7.49'
+		puts 'Mutton curry   £6.59'
+		puts 'Naan bread     £1.29'
+		puts 'Poppadom       £0.75'
+	end
+
+	def select(item)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for select is too high. [20.64/15]